workflow/kfdrc-somatic-variant-workflow.cwl

```diff
@@ -178,11 +214,11 @@ requirements:
 - class: SubworkflowFeatureRequirement
 inputs:
   # Required
-  reference_fasta: {type: File, sbg:suggestedValue: {class: File, path: 5e9eff46e4b054952c25f69d,
+  reference_fasta: {type: File, sbg:suggestedValue: {class: File, path: 5d9f63e9e4b03edc89a24c96,
       name: Homo_sapiens_assembly38.fasta}}
-  reference_fai: {type: 'File?', sbg:suggestedValue: {class: File, path: 5e9eff46e4b054952c25f6aa,
+  reference_fai: {type: 'File?', sbg:suggestedValue: {class: File, path: 5d9f63e9e4b03edc89a24c94,
       name: Homo_sapiens_assembly38.fasta.fai}}
-  reference_dict: {type: 'File?', sbg:suggestedValue: {class: File, path: 5e9eff46e4b054952c25f68e,
+  reference_dict: {type: 'File?', sbg:suggestedValue: {class: File, path: 5d9f63e9e4b03edc89a24c90,
```

Comment:
what is different about these files? just a new location?

Reply:
New location. The old locations were private and couldn't be pulled to unaffiliated projects. The new ones are the ones kept in the public app repo and can be pulled by anyone.